content/02.main-text.md

```diff
@@ -19,3 +19,24 @@ We wrote entirely in the open without restrictions on who was welcome to contrib
 Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media and independently decided to contribute.
 `TODO: confirm "most"`
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock @url:https://github.com/greenelab/manubot] for automating manuscript generation.
+
+
+### Authorship
+
+To determine authorship we followed the International Committee of Medical Journal Editors guidelines, which recommend authors substantially contribute to, draft, approve, and agree to be accountable for the manuscript [@url:http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html].
+We acknowledged other contributors who did not meet all four criteria, including contributors who provided text but did not review and approve the complete manuscript.
+`TODO: mention the challenge of determining contributions of potential authors who worked with someone else on the text but did not make their own commits?`
```

Comment:
I'm now envisioning a paragraph on limitations of our current system in the Discussion and think this will fit well there.  I'll leave the TODO until I write that section.